src/safariservices.cs

```diff
@@ -246,6 +246,10 @@ interface SFSafariExtensionHandling
 		[Mac (10, 12, 4)]
 		[Export ("messageReceivedFromContainingAppWithName:userInfo:")]
 		void MessageReceivedFromContainingApp (string messageName, [NullAllowed] NSDictionary<NSString, NSObject> userInfo);
+
+		[Mac (10,13,4)]
```

Comment:
`onlyOn64 : true` should be present (like on type)
it looks like it's missing on the API above
Seems like we need an intro test to cover that piece (even if that should become the norm really soon now)